lighthouse-core/report/html/renderer/report-ui-features.js

```diff
@@ -114,11 +111,61 @@ class ReportUIFeatures {
     // There is only a sticky header when at least 2 categories are present.
     if (Object.keys(this.json.categories).length >= 2) {
       this._setupStickyHeaderElements();
-      this._document.addEventListener('scroll', this._updateStickyHeaderOnScroll);
-      window.addEventListener('resize', this._updateStickyHeaderOnScroll);
+      const containerEl = this._dom.find('.lh-container', this._document);
+      const elToAddScrollListener = this._getScrollParent(containerEl);
+      elToAddScrollListener.addEventListener('scroll', this._updateStickyHeaderOnScroll);
+      // We can't rely on listening to the window resize event for DevTools, so we first
+      // attempt the new ResizeObserver web platform feature. It has poor cross browser
+      // support, so we should check that it's supported. However, there are some performance
+      // issues with using window.ResizeObserver - it updates much more often than the 'resize'
```

Comment:
When using the ResizeObserver in the deploy app (It's disabled outside devtools but if you revert that bit), and add some logging to the callback:

```
console.log(topbarBottom, scoreScaleTop, showStickyHeader);
```

![image](https://user-images.githubusercontent.com/4071474/58600744-a9344580-823a-11e9-9fc0-592ebd08f528.png)


You see that the position of our elements are jumping all over the place!

Reply:
So we should just disable it outside DevTools for now, luckily it works there perfectly.